cosmic/sample/sampler/independent.py

```diff
@@ -408,12 +409,12 @@ def sample_porb(self, mass1, mass2, ecc, model='sana12', size=None):
             porb = porb_yr*yr_day
         elif model == 'sana12':
             from cosmic.utils import rndm
-            porb = 10**rndm(a=0.15, b=5.5, g=0.55, size=size)
+            porb = 10**rndm(a=0.15, b=5.5, g=0.45, size=size)
         elif model == 'renzo19':
             from cosmic.utils import rndm
             porb = 10**(np.random.uniform(0.15, 5.5, size))
             ind_massive, = np.where(mass1 > 15)
-            porb[ind_massive] = 10**rndm(a=0.15, b=5.5, g=0.55, size=len(ind_massive))
+            porb[ind_massive] = 10**rndm(a=0.15, b=5.5, g=0.45, size=len(ind_massive))
```

Comment:
Same as comment above. Maybe you're mixing this up with the eccentricity power law (\eta =  0.45)?